src/modules/useDatastore.js

```diff
@@ -0,0 +1,21 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import { GENERAL_SETTINGS, NAMESPACE } from '../constants/data-store'
+
+/**
+ * Query to get General Settings from Data store
+ * */
+
+export const getGeneralKeyQuery = {
+    generalSettings: {
+        resource: `dataStore/${NAMESPACE}/${GENERAL_SETTINGS}`,
+    },
+}
+
+export const useGeneralDataStore = () => {
```

Comment:
Would it work to just use `useDataQuery` directly in the component, rather than wrapping it in another hook?